libkbfs/conflict_resolver.go

```diff
@@ -220,13 +226,29 @@ func (cr *ConflictResolver) getMDs(ctx context.Context, lState *lockState) (
 	if err != nil {
```

Comment:
it's quite easy to forget that `get{M,Unm}ergedMDUpdates` returns a read-only list. Perhaps we should make those functions take something like `mdReqType` or `blockReqType` to signal that we want copies? Perhaps as a TODO?